ansible_collections/arista/avd/molecule/cv_deploy/workspace_force_false.yml

```
@@ -0,0 +1,76 @@
+---
+- name: Converge - cv_submit_workspace_force = false
+  hosts: SITE1_FABRIC
+  connection: local
+  gather_facts: false
+  vars:
+    cv_server: www.cv-staging.corp.arista.io
+    cv_token: "{{ lookup('env', 'CVAAS_AAWG_CI') }}"
+    cv_verify_certs: false
+    cv_skip_missing_devices: true
+    eos_config_dir: "{{ playbook_dir }}/intended/configs/test_configs"
+    structured_dir: "{{ playbook_dir }}/intended/structured_configs/test_configs"
+    intended_tag_device: avd-ci-leaf1
+    intended_tags: "{{ lookup('file', structured_dir ~ '/' ~ intended_tag_device ~ '.yml')| from_yaml }}"
+
+  tasks:
+    - name: Generate random string
+      ansible.builtin.set_fact:
+        r: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=4') }}"
+      run_once: true
+
+    - name: Set facts
+      ansible.builtin.set_fact:
+        cv_workspace_name: avd-cv-deploy-{{ r }}
+        cv_workspace_description: sample description
+        cv_change_control_name: cc_cv_deploy-{{ r }}
+        cv_change_control_description: sample description
+        cv_register_detailed_results: true
+      run_once: true
```

Although not needed if this play is run after 'workspace_force_true' (as updated inventory will be inherited by this play as well) - adding it here to be able to run 'workspace_force_false' when 'workspace_force_true' is excluded from converge workflow
```suggestion
      run_once: true

    - name: Dynamically add inactive device avd-ci-core1 to group INACTIVE
      run_once: true
      add_host:
        name: avd-ci-core1
        groups: INACTIVE
```